pkg/serviceaccounts/controllers/docker_registry_service.go

```diff
@@ -48,7 +48,7 @@ type serviceLocation struct {
 
 var serviceLocations = []serviceLocation{
 	{namespace: "default", name: "docker-registry"},
-	{namespace: "openshift-image-registry", name: "registry"},
+	{namespace: "openshift-image-registry", name: "image-registry"},
```

Comment:
@deads2k why doesn't this controller/logic use the same registry hostname retriever logic as everything else?  (ie lookup the value from internalRegistryHostname, fall back to looking for existing services).